src/app/(main)/page.tsx

```diff
@@ -0,0 +1,10 @@
+import type { Metadata } from "next"
+import { IndexPage } from "../../components/index-page"
+
+export const metadata: Metadata = {
+  title: "GraphQL | A query language for your API",
```

Comment:
Not related to the PR, but I just wanna take some notes while I'm reading it. Fine with me to turn them into issues.

We could move `description: GraphQL is a query language for APIs and a runtime for fulfilling those queries with your existing data.` from `next/head` inside of index-page.tsx to here.